src/spikeinterface/core/baserecording.py

```diff
@@ -746,6 +746,30 @@ def _select_segments(self, segment_indices):
 
         return SelectSegmentRecording(self, segment_indices=segment_indices)
 
+    def get_channel_locations(
+        self,
+        channel_ids: list | np.ndarray | tuple | None = None,
+        axes: "xy" | "yz" | "xz" = "xy",
+    ) -> np.ndarray:
+        """
+        Get the physical locations of specified channels.
+
+        Parameters
+        ----------
+        channel_ids : array-like, optional
+            The IDs of the channels for which to retrieve locations. If None, retrieves locations
+            for all available channels. Default is None.
+        axes : str, optional
```

Comment:
```suggestion
        axes : "xy" | "yz" | "xz" | "xyz", default: "xy"
```

I know we need to rehash the `None`/ `optional` argument with the fuller team, but I think it was not being debated to do this part this way